site/src/component/SearchPagination/SearchPagination.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, useEffect } from "react";
+import { Pagination } from "react-bootstrap";
+import { PAGE_SIZE } from "src/helpers/constants";
+import { useAppDispatch, useAppSelector } from "src/store/hooks";
+import { setPageNumber } from "src/store/slices/searchSlice";
+import { SearchIndex } from "src/types/types";
+
+interface SearchPaginationProps {
+  index: SearchIndex;
+}
+
+const SearchPagination: FC<SearchPaginationProps> = ({ index }) => {
```

Comment:
Can you add some quick comments on this module explaining what it does and what some of the key functions do?